### PR TITLE
Update to latest version of URL, use new install mechanism

### PR DESCRIPTION
**Pull request by triblondon**
See #764.

### CODE REVIEW

**Comment by JakeChampion** [comment]
This seems to fail in IE 8/12/13 :

``` bash
 - ie 13 (Sauce results: https://saucelabs.com/tests/842735ead22d42cf9eaeb9424189bf2d)
    -> URL       http://127.0.0.1:3000/test/tests?mode=targeted&feature=URL
 - ie 12 (Sauce results: https://saucelabs.com/tests/49bbe4a2c26f4bedb589c07c2d18b30f)
    -> URL       http://127.0.0.1:3000/test/tests?mode=targeted&feature=URL
 - ie 8 (Sauce results: https://saucelabs.com/tests/99688b1ce0624bd1931ad3cd6e75cd95)
    -> URL       http://127.0.0.1:3000/test/tests?mode=targeted&feature=URL
```

**Comment by triblondon** [comment]
Yep, will look at those.  I lazily relied on the PR to run CI rather than running it from my local machine first.

**Comment by triblondon** [comment]
@JakeChampion success! I had to disable number of tests because of buggy native implementations in Firefox and edge, and because some of the features tested are not actually implemented by the polyfill.  But I also added a slew of extra tests by updating from the upstream repo. Ended up removing edge targeting because native support is almost perfect in edge, and adding iOS Safari targeting because URL is not supported on Safari on any platform.

Could you merge if you're happy with this, and tag for whatever your next release is?

File: polyfills/URL/tests.js [hunk 1]
```diff
@@ -1,28 +1,35 @@
 /* eslint-env mocha, browser*/
 /* global proclaim, it */
 
+function toArray(it) {
```
Comment (JakeChampion) [inline polyfills/URL/tests.js:4]: The tests which use this function are commented out, should we comment this out or remove it?

Comment (triblondon) [reply polyfills/URL/tests.js:4]: I wanted to make as few changes to the tests as possible.  I don't really mind either of those options, or just leaving it as-is.

**Comment by rowanmanning** [comment]
This looks good to me 👍

**Pull request merged**